src/demo/helpers.js

```diff
@@ -171,9 +171,11 @@ export const finishMovement = (t, pause = true) => {
   });
 };
 
-export const resetTraining = () => {
+export const resetTraining = state => {
+  const trainer = state.trainer;
+  trainer.clearAll();
   setState({
-    trainer: null,
+    trainer,
```

Comment:
do we still need to re-set `state.trainer` here since we're no longer setting it to `null`?